torch/_dynamo/symbolic_convert.py

```diff
@@ -1747,36 +1828,65 @@ def create_call_resume_at(self, inst):
             for k in self.symbolic_locals.keys()
             if k in reads and k not in self.cell_and_freevars()
         )
-        nargs = len(self.stack) + len(argnames)
+
+        cg = PyCodegen(self)
+
+        # Python does not allow null to be an arg to a function, so
+        # we remove nulls from the stack and restore them in the
+        # prologue of the resume function
+        null_idxes: List[int] = []
+        if sys.version_info >= (3, 11):
+            nop_name = unique_id(_nop_for_pop_null.__code__.co_qualname)
+
+            self.output.install_global(
+                nop_name,
+                types.FunctionType(
```

Comment:
Do we actually need to create a new function here?  I think we could just load a pointer to a function defined in dynamo (similar to stuff like rotn helpers).